docs/proposals/1374-multi-cluster-inference/README.md

```diff
@@ -0,0 +1,613 @@
+    IGB["Inference Gateway<br/>(dataplane)"]
+    IPL["InferencePool<br/>(export annotation)"]
+  end
+
+  %% Controller relationships (control plane)
+  MCI -. "creates/updates" .-> IPIMP
+  IGCTL -. "watches" .-> IPIMP
+  IGCTL -. "configure" .-> IGB
+
+  %% Side-channel (control) consultations to EPP (dashed)
+  IGB -. "gRPC ext-proc (EndpointMode)" .-> EPPA
+  PGA -. "gRPC ext-proc" .-> EPPA
+
+  %% HTTP data paths (solid)
+  CLIENT --> IGB
+
+  %% Option A: EndpointMode
+  IGB -- "HTTP (EndpointMode)" --> PODA
+
+  %% Option B: ParentMode (via remote parent gateway)
+  IGB -- "HTTP (ParentMode)" --> PGA
+  PGA -- "HTTP" --> PODA
+
+  %% (Visual grouping / hints)
+  IPA --- PODA
+  IPA --- EPPA
+```
+
+### Exporting Cluster (Cluster A) Manifests
+
+In this example, Cluster A exports the InferencePool to all clusters in the Cluster set using `EndpointMode`. This will
+cause the MCI controller to create an InferencePoolImport resource in all clusters except the exporting cluster.
+
+```yaml
+# Export the pool by annotation
+apiVersion: inference.networking.k8s.io/v1
+kind: InferencePool
+metadata:
+  name: llm-pool
+  namespace: example
+  annotations:
+    inference.networking.x-k8s.io/export: "ClusterSet"
+    inference.networking.x-k8s.io/export-mode: "EndpointMode"   # or "ParentMode"
+spec:
+  endpointPickerRef:
+    name: epp
+    portNumber: 9002
+  selector:
+    matchLabels:
+      app: my-model
+  targetPorts:
+  - number: 8080
+---
+# EPP exposed via LoadBalancer for simplicity; ClusterIP also works with podCIDR reachability or via parents
+apiVersion: v1
+kind: Service
+metadata:
+  name: epp
+  namespace: example
+spec:
+  selector:
+    app: epp
+  ports:
+  - name: ext-proc
+    port: 9002
+    targetPort: 9002
+    appProtocol: http2
+  - name: health
+    port: 9003
+    targetPort: 9003
+  - name: metrics
+    port: 9090
+    targetPort: 9090
+  type: LoadBalancer
+```
+
+### Importing Cluster (Cluster B) Manifests
+
+In this example, the InferencePlatform Owner has configured an HTTPRoute to route to endpoints of the Cluster A InferencePool
+by referencing the InferencePoolImport as a `backendRef`. The parent IG(s) of the HTTPRoute are responsible for routing to the
+endpoints selected by the EPP referenced by the exported InferencePool (surfaced through InferencePoolImport status).
+
+The InferencePoolImport is controller-managed; shown here only to illustrate the expected status shape.
+
+```yaml
+apiVersion: inference.networking.x-k8s.io/v1alpha1
+kind: InferencePoolImport
+metadata:
+  name: llm-pool      # mirrors exporting pool name
+  namespace: example  # mirrors exporting pool namespace
+status:
+  clusters:
+  - name: cluster-a
+    routingMode: EndpointMode # or ParentMode
+    targetPortNumber: 8080
+    endpointPicker:
+     name: epp
+     service:
+       type: LoadBalancer
+       addresses:
+       - 1.2.3.4          # EPP service address (IP or hostname)
```

Comment:
What if we just flatten this to a list of addresses for the EPP and a potential type field? IE the type may be a hostname, an IP, or something else entirely. In some cases these addresses would point directly to Pod IPs, in other cases these would point to a LB IP or hostname. 

We could also consider some kind of reference to an EndpointSlice, but since I'd expect the number of EPP endpoints to often be 1-2, with a max of ~5, communicating that in status feels like it could be a bit simpler.